docs/eventlisteners.md

```diff
@@ -14,6 +14,35 @@ references a listener pod. This listener pod accepts the incoming events and
 does what has been specified in the corresponding
 `TriggerBinding`s/`TriggerTemplate`s.
 
+
+## Parameters
+`EventListener`s can provide `params` which are merged with the `TriggerBinding`
+`params` and passed to the `TriggerTemplate`. Each parameter has a `name` and a
+`value`.
+
+
+## Event Interceptors
+
+Triggers within an EventListener can optionally specify an interceptor field
+which contains an `ObjectReference` to a Kubernetes Service. If an interceptor 
+is specified, the `EventListener` sink will forward incoming events to the 
+service referenced by the interceptor over HTTP. The service is expected to 
```

Comment:
I didn't even catch the HTTP part when reading this initially. Maybe still be explicit about port 80?

Reply:
Yeah, the port 80 thing is true at the moment but should change once we support all Addressable types. Will add to the docs for now